### PR TITLE
Pino 7 Type Definition

### PR DESCRIPTION
**Pull request by jarcodallo**
Fix for #175

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import { IncomingMessage, ServerResponse } from 'http';
+import pino from 'pino';
+
+export = PinoHttp;
+
+declare function PinoHttp(opts?: PinoHttp.Options, stream?: pino.DestinationStream): PinoHttp.HttpLogger;
+declare function PinoHttp(stream?: pino.DestinationStream): PinoHttp.HttpLogger;
+
+declare namespace PinoHttp {
```
Comment (kibertoad) [inline index.d.ts:9]: AFAIR, namespaces are no longer a recommended TS pattern, using module is preferable.

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import { IncomingMessage, ServerResponse } from 'http';
+import pino from 'pino';
+
+export = PinoHttp;
```
Comment (kibertoad) [inline index.d.ts:4]: Can you do named exports instead? This style of export has issues when esModuleInterop is disabled, AFAIR.

File: index.d.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import { IncomingMessage, ServerResponse } from 'http';
```
Comment (kibertoad) [inline index.d.ts:1]: looks like you need to add node ts types as dev dependency

**Comment by rstrand** [comment]
These types look copied from definitely typed (and updated/fixed) but with the attribution removed from the top of index.d.ts
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/pino-http/index.d.ts

**Comment by kibertoad** [comment]
@jarcodallo Could you please restore the attribution?

**Comment by jarcodallo** [comment]
> These types look copied from definitely typed (and updated/fixed) but with the attribution removed from the top of index.d.ts https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/pino-http/index.d.ts

Yeah I copy the definition from there and add mirror changes. I will restore the attribution.

**Comment by jarcodallo** [comment]
> @jarcodallo Could you please restore the attribution?

I will, sorry for that. I just remove all the comments

**Review by kibertoad — APPROVED** [review]

**Pull request merged**